core/domain/feedback_services_test.py

```diff
@@ -64,4 +65,169 @@ def test_status_of_newly_created_thread_is_open(self):
             EXP_ID, 'a_state_name', None, 'a subject', 'some text')
         threadlist = feedback_services.get_threadlist(EXP_ID)
         thread_status = threadlist[0]['status']
-        self.assertEqual(thread_status, feedback_models.STATUS_CHOICES_OPEN)   
+        self.assertEqual(thread_status, feedback_models.STATUS_CHOICES_OPEN)
+
+    def _generate_thread_id(self, exp_id):
```

Comment:
Also, please make sure to run _all_ tests before any commit, and update the backend test count.